modules/pubwiseAnalyticsAdapter.js

```diff
@@ -54,72 +73,268 @@ function enrichWithUTM(dataBag) {
   try {
     for (let prop in utmKeys) {
       utmKeys[prop] = utils.getParameterByName(prop);
-      if (utmKeys[prop] != '') {
+      if (utmKeys[prop]) {
         newUtm = true;
         dataBag[prop] = utmKeys[prop];
       }
     }
 
     if (newUtm === false) {
       for (let prop in utmKeys) {
-        let itemValue = storage.getDataFromLocalStorage(`pw-${prop}`);
-        if (itemValue.length !== 0) {
+        let itemValue = localStorage.getItem(setNamespace(prop));
```

Comment:
why you decided to remove prebid storageManager ?

Reply:
This was a regression. The next update will have it re-added.